api/issue.go

```diff
@@ -144,8 +144,10 @@ type UpdateSchemaDetail struct {
 	DatabaseName string `json:"databaseName"`
 	// Statement is the statement to update database schema.
 	Statement string `json:"statement"`
-	// Statement is the rollback statement of the statement.
+	// RollbackStatement is the rollback statement of the statement.
 	RollbackStatement string `json:"rollbackStatement"`
+	// EarliestAllowedTs the earliest execution time of the change at system local Unix timestamp in nanoseconds.
+	EarliestAllowedTs int64 `jsonapi:"attr,earliestAllowedTs"`
```

Comment:
I may need more background info.

This field is updated by `TaskPatch` now, will all relevant logics be refactored?
In my opionion, `EarliestAllowedTs` is a Task level field, and `schema change` is only ONE kind of task. If we do apply such refactor, need we refactor them also (like we can specified `EarliestAllowedTs` for database creation task now) ?

Reply:
I see. I misread your intention.